src/app/routes/api.routes.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express'
+
+import { ADMIN_NESTED_ROUTE, AdminRouter } from './admin'
+
+export const API_V3_ROOT_ROUTE = '/api/v3'
+
+export const ApiRouter = Router()
+
+ApiRouter.use(ADMIN_NESTED_ROUTE, AdminRouter)
```

Comment:
Would it be simpler to just do this? The `AdminRouter` doesn't need to know where it's mounted, that's decided higher up in the hierarchy in the `ApiRouter`.

```typescript
ApiRouter.use('/admin', AdminRouter)
```

Reply:
Then we can cut out variables such as `ADMIN_NESTED_ROUTE`

Reply:
Makes sense. Should we still give it an `ADMIN_ROOT_ROUTE` constant then? Or only do that when we need to make it a constant?

Reply:
I understand now, the route should not be exposed by the router but set by the caller. Will remove all exported route constants and set them in the router